src/adapter/src/coord/ddl.rs

```diff
@@ -562,6 +600,24 @@ impl Coordinator {
                 }
                 self.drop_compute_sinks(sink_ids);
             }
+            if !copy_to_sinks_to_drop.is_empty() {
+                let mut sink_ids = Vec::new();
+                for (_conn_id, dropped_name, sink_id) in copy_to_sinks_to_drop {
+                    if let Some(copy_to) = self.remove_active_copy_to(sink_id.global_id).await {
+                        self.retire_execution(
+                            StatementEndedExecutionReason::Canceled,
+                            Default::default(), // TODO (mouli): is ctx_extra required here
+                        );
+                        let _ = copy_to.sender.expect("oneshot channel exists").send(
+                            CopyToResponseUnary::Error(format!(
+                                "query could not complete because {dropped_name} was dropped"
+                            )),
+                        );
+                    }
+                    sink_ids.push(sink_id);
+                }
+                self.drop_compute_sinks(sink_ids);
```

Comment:
This whole block seems uncovered?

Reply:
let me add more tests

Reply:
Added a test which should now test out this part.